src/ResourceManager/Media/Commands.Media.Test/ScenarioTests/MediaTests.ps1

```diff
@@ -115,7 +115,7 @@ function Test-Media
 
   <# Create a media service with multiple storage accounts #>
   $tags = @{"tag1" = "value1"; "tag2" = "value2"}
-  $mediaService = New-AzureRmMediaService -ResourceGroupName $rgname -AccountName $accountName -Location $location -StorageAccounts $storageAccounts -Tags $tags
+  $mediaService = New-AzureRmMediaService -ResourceGroupName $rgname -AccountName $accountName -Location $location -StorageAccounts $storageAccounts -Tag $tags
```

Comment:
why is this change in this PR?

Reply:
Similar to MachineLearning, the Media tests weren't being run during the build, but I was able to make a few changes to get them to run successfully on the build. This included:
- Changing the `-Tags` parameter to `-Tag`
- Changing the type of ResourceManagementClient instantiated in the test controller